presto-hive/src/main/java/com/facebook/presto/hive/HiveSplitManager.java

```diff
@@ -155,30 +155,29 @@ public HiveSplitManager(
     public ConnectorSplitSource getSplits(ConnectorTransactionHandle transaction, ConnectorSession session, ConnectorTableLayoutHandle layoutHandle, SplitSchedulingStrategy splitSchedulingStrategy)
     {
         HiveTableLayoutHandle layout = (HiveTableLayoutHandle) layoutHandle;
+        SchemaTableName tableName = layout.getSchemaTableName();
```

Comment:
Would the code change in `HiveSplitManager`  better suits for commit `Cleanup code in HiveSplitManager` ? :)

Reply:
Sure, let me change it so this commit has the minimal changes to add the table name to the handle.